numba/_dispatcher.cpp

```diff
@@ -1612,6 +1620,17 @@ static PyTypeObject DispatcherType = {
 };
 
 
+#if (PY_MAJOR_VERSION >= 3) && ((PY_MINOR_VERSION == 12) || (PY_MINOR_VERSION == 13))
+static
+bool is_sysmon_enabled(PyObject * self) {
+    if (PyObject_IsInstance(self, (PyObject*)&DispatcherType)) {
+        Dispatcher* disp = (Dispatcher*)self;
+        return disp->enable_sysmon;
+    }
+    return false;
```

Comment:
this should probably raise a type error?

Reply:
Removed type check.
Base branch changes made the type check obsolete.

Reply:
Ah yes, that makes the code much more simple!